pipelines/template/settings.py

```diff
@@ -0,0 +1 @@
+"""Template Pipeline settings and constants"""
```

Comment:
do we still need py.typed? is typing optional or not?

Reply:
until I have touched all pipelines, typing will stay optional, then we can set it to mandatory across the repo